### PR TITLE
Bump turbo in examples.

### PR DESCRIPTION
**Pull request by anthonyshew**
Bumping `turbo` to latest in examples. 🥳

### CODE REVIEW

File: examples/basic/package.json [hunk 1]
```diff
@@ -1,20 +1,19 @@
 {
-  "name": "my-turborepo",
```
Comment (mehulkar) [inline examples/basic/package.json:2]: Bring the name back?

File: examples/basic/package.json [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-    "@repo/typescript-config": "workspace:*",
-    "prettier": "^3.1.0",
-    "turbo": "^1.10.16"
+    "@changesets/cli": "^2.25.2",
+    "prettier": "^3.0.3",
+    "turbo": "^1.11.1"
   },
-  "packageManager": "pnpm@8.9.0",
-  "engines": {
-    "node": ">=18"
```
Comment (mehulkar) [inline examples/basic/package.json:18]: Keep engines?

File: examples/kitchen-sink/package.json [hunk 3]
```diff
@@ -2,19 +2,18 @@
...
   "devDependencies": {
-    "prettier": "^3.1.0",
-    "turbo": "^1.10.16"
+    "@changesets/cli": "^2.25.2",
+    "prettier": "^3.0.3",
+    "turbo": "^1.11.1"
   },
-  "packageManager": "pnpm@8.9.0",
-  "engines": {
-    "node": ">=18"
```
Comment (mehulkar) [inline examples/kitchen-sink/package.json:18]: keep this

File: examples/with-react-native-web/package.json [hunk 4]
```diff
@@ -1,21 +1,19 @@
 {
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*"
-  ],
```
Comment (mehulkar) [inline examples/with-react-native-web/package.json:6]: did this move to pnpm in this PR?

**Review by mehulkar — COMMENTED** [review]
Looks like a bunch of stuff other than updating turbo, but 🚢 if you feel good about it I guess?

File: examples/with-vue-nuxt/pnpm-lock.yaml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```
Comment (mehulkar) [inline examples/with-vue-nuxt/pnpm-lock.yaml:1]: downgrade?

**Review by mehulkar — APPROVED** [review]
One question, but otherwise 🚢

File: examples/with-vue-nuxt/pnpm-lock.yaml (hunk 5 above)
Comment (anthonyshew) [reply examples/with-vue-nuxt/pnpm-lock.yaml:1]: I, uh, don't have an explanation there. I'm going to let pnpm lead the way.

**Pull request merged**